.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
   hooks:
   - id: black
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.4.2
+  rev: v0.4.8
```

Comment:
This changes are already available in https://github.com/SatelliteQE/airgun/pull/1383